app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/PaymentsErrorIcon.kt

```diff
@@ -0,0 +1,73 @@
+package com.aptoide.android.aptoidegames.drawables.icons
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.aptoide.android.aptoidegames.theme.error
+
+@Preview
+@Composable
+fun TestPaymentsErrorIcon() {
+  Image(
+    imageVector = getPaymentsErrorIcon(),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getPaymentsErrorIcon(): ImageVector = ImageVector.Builder(
+  name = "PaymentsErrorIcon",
+  defaultWidth = 64.dp,
+  defaultHeight = 64.dp,
+  viewportWidth = 64f,
+  viewportHeight = 64f,
+).apply {
+  path(
+    fill = SolidColor(error),
```

Comment:
Let's not hardcode here but provide it through arguments.
Better set one of the new colours in the place it is used.

Reply:
error views have a different ticket. I just added them for the project to compile